### DISCUSSION TITLE
Automatically download of pictures

### DISCUSSION THREAD

**Discussion by Jogge** [scaarup/aula#50] [Ideas]
Our kindergarten creates a picture album for each month. A few days after change of month, the kindergarten deletes all pictures for the previous month.

This only gives us a few days to download the latest uploaded pictures, if we want to save these... It could be a very useful feature, if this integration could download all pictures, so we don't miss any pictures where our children are present.

To a start the pictures could be stored locally in a Home Assistant folder. But the perfect solution in my opinion would download all pictures and upload the pictures to Google Drive.

The pictures could also be used in Card on the Dashboard that is showing the latest pictures in a slideshow.

**Comment by scaarup** [comment]
I'm not completely convinced that this belongs here :)

Sounds like you should create a script and run that periodically.
